pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -509,3 +513,41 @@ func (r *Reconciler) markUpgrade(ctx context.Context, tc *v1alpha1.TektonChain)
 	}
 	return v1alpha1.RECONCILE_AGAIN_ERR
 }
+
+func AddControllerEnv(controllerEnvs []v1alpha1.ControllerEnv) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "Deployment" {
+			return nil
+		}
+
+		d := &appsv1.Deployment{}
+		err := runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, d)
+		if err != nil {
+			return err
+		}
+
+		if controllerEnvs != nil {
```

Comment:
with the above command we can remove this `if` block.
and If I remember correctly, `range` is `nil` safe.